shell/components/x-list.js

```diff
@@ -8,11 +8,16 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import XTemplate from './xen-template.js';
-import XElement from './xen-element.js';
-import XState from './xen-state.js';
+import Xen from './xen/xen.js';
 
-class XList extends XState(XElement) {
+let HTMLElement;
+if (typeof window == 'undefined') {
```

Comment:
nit: I usually see `===` for these?

Reply:
When testing against a literal string, there is no difference between `==` and `===`, but I can change it if it's bothersome.